From e8cdbf589cfd4e74f3097d9f050d9aa60543e2f2 Mon Sep 17 00:00:00 2001
From: Alexei Takaseev <taf@altlinux.org>
Date: Fri, 26 Dec 2025 19:54:03 +0800
Subject: Update path for socket and log


diff --git a/src/include/pool.h b/src/include/pool.h
index ea6f87e12..40f5dd574 100644
--- a/src/include/pool.h
+++ b/src/include/pool.h
@@ -58,13 +58,13 @@
 #define HBA_CONF_FILE_NAME "pool_hba.conf"
 
 /* pid file directory */
-#define DEFAULT_LOGDIR "/tmp"
+#define DEFAULT_LOGDIR "/var/log/pgpool"
 
 /* Unix domain socket directory */
-#define DEFAULT_SOCKET_DIR "/tmp"
+#define DEFAULT_SOCKET_DIR "/var/run/pgpool"
 
 /* Unix domain socket directory for watchdog IPC */
-#define DEFAULT_WD_IPC_SOCKET_DIR "/tmp"
+#define DEFAULT_WD_IPC_SOCKET_DIR "/var/run/pgpool"
 
 /* pid file name */
 #define DEFAULT_PID_FILE_NAME "/var/run/pgpool/pgpool.pid"
diff --git a/src/sample/pgpool.conf.sample-stream b/src/sample/pgpool.conf.sample-stream
index ba6b923b0..0f48618d7 100644
--- a/src/sample/pgpool.conf.sample-stream
+++ b/src/sample/pgpool.conf.sample-stream
@@ -40,7 +40,7 @@ backend_clustering_mode = streaming_replication
 #port = 9999
                                    # Port number
                                    # (change requires restart)
-#unix_socket_directories = '/tmp'
+unix_socket_directories = '/var/run/pgpool'
                                    # Unix domain socket path(s)
                                    # The Debian package defaults to
                                    # /var/run/postgresql
@@ -67,7 +67,7 @@ backend_clustering_mode = streaming_replication
 #pcp_port = 9898
                                    # Port number for pcp
                                    # (change requires restart)
-#pcp_socket_dir = '/tmp'
+pcp_socket_dir = '/var/run/pgpool'
                                    # Unix domain socket path(s) for pcp
                                    # The Debian package defaults to
                                    # /var/run/postgresql
@@ -312,7 +312,7 @@ backend_clustering_mode = streaming_replication
 
 # -- Only used if logging_collector is on ---
 
-#log_directory = '/tmp/pgpool_logs'
+log_directory = '/var/log/pgpool'
                                         # directory where log files are written,
                                         # can be absolute
 #log_filename = 'pgpool-%Y-%m-%d_%H%M%S.log'
