diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/advanced.sgml pgpool-II-4.6.0/doc/src/sgml/advanced.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/advanced.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/advanced.sgml	2025-03-05 04:29:30.498067815 +0000
@@ -158,10 +158,10 @@
    </para>
   </sect2>
 
-  <sect2 id="tutorial-watchdog-changing-automatic-register-in-recovery">
+  <sect2 id="changing-automatic-register-in-recovery">
    <title>Automatic registration of a server as a standby in recovery</title>
 
-   <indexterm zone="tutorial-watchdog-changing-automatic-register-in-recovery">
+   <indexterm zone="changing-automatic-register-in-recovery">
     <primary>WATCHDOG</primary>
    </indexterm>
    <para>
@@ -211,7 +211,7 @@
   </sect2>
  </sect1>
 
- <sect1 id="tutorial-watchdog-integrating-external-lifecheck">
+ <sect1 id="integrating-external-lifecheck">
   <title>Integrating external lifecheck with watchdog</title>
 
   <para>
@@ -402,10 +402,10 @@
     also uses the same channel and technique.
    </para>
 
-   <sect3 id="tutorial-watchdog-external-lifecheck-get-nodes">
+   <sect3 id="external-lifecheck-get-nodes">
     <title>Getting list of configured watchdog nodes</title>
 
-    <indexterm zone="tutorial-watchdog-external-lifecheck-get-nodes">
+    <indexterm zone="external-lifecheck-get-nodes">
      <primary>WATCHDOG</primary>
     </indexterm>
     <para>
@@ -529,9 +529,9 @@
    </para>
   </sect2>
 
-  <sect2 id="tutorial-watchdog-restrictions-down-watchdog-mode">
+  <sect2 id="restrictions-down-watchdog-mode">
    <title>Connecting to <productname>Pgpool-II</productname> whose watchdog status is down</title>
-   <indexterm zone="tutorial-watchdog-restrictions-down-watchdog-mode">
+   <indexterm zone="restrictions-down-watchdog-mode">
     <primary>WATCHDOG</primary>
    </indexterm>
    <para>
@@ -543,9 +543,9 @@
    </para>
   </sect2>
 
-  <sect2 id="tutorial-watchdog-restrictions-down-watchdog-require-restart">
+  <sect2 id="restrictions-down-watchdog-require-restart">
    <title><productname>Pgpool-II</productname> whose watchdog status is down requires restart</title>
-   <indexterm zone="tutorial-watchdog-restrictions-down-watchdog-require-restart">
+   <indexterm zone="restrictions-down-watchdog-require-restart">
     <primary>WATCHDOG</primary>
    </indexterm>
    <para>
@@ -555,9 +555,9 @@
    </para>
   </sect2>
 
-  <sect2 id="tutorial-watchdog-restrictions-active-take-time">
+  <sect2 id="restrictions-active-take-time">
    <title>Watchdog promotion to active takes few seconds</title>
-   <indexterm zone="tutorial-watchdog-restrictions-active-take-time">
+   <indexterm zone="restrictions-active-take-time">
     <primary>WATCHDOG</primary>
    </indexterm>
    <para>
@@ -701,7 +701,7 @@
      to interact with watchdog.
     </para>
     <para>
-     See <xref linkend="tutorial-watchdog-integrating-external-lifecheck"> for details
+     See <xref linkend="integrating-external-lifecheck"> for details
       on how to use watchdog IPC interface for integrating external/3rd party systems.
     </para>
    </sect3>
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/connection-settings.sgml pgpool-II-4.6.0/doc/src/sgml/connection-settings.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/connection-settings.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/connection-settings.sgml	2025-03-05 04:29:49.844140656 +0000
@@ -1525,7 +1525,7 @@
       </programlisting>
       The <xref linkend="guc-recovery-1st-stage-command"> should
       generate the parameter. See <xref
-      linkend="example-cluster-pgpool-config-online-recovery"> for a
+      linkend="pgpool-config-online-recovery"> for a
       complete example of the command.
      </para>
      <para>
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/example-cluster.sgml pgpool-II-4.6.0/doc/src/sgml/example-cluster.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/example-cluster.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/example-cluster.sgml	2025-03-05 04:33:48.538042873 +0000
@@ -683,7 +683,7 @@
    </programlisting>
   </sect3>
 
-  <sect3 id="example-cluster-pgpool-config-listen-addresses">
+  <sect3 id="pgpool-config-listen-addresses">
    <title>listen_addresses</title>
    <para>
     To allow Pgpool-II and PCP to accept all incoming connections, set the following
@@ -741,7 +741,7 @@
    </programlisting>
   </sect3>
 
-  <sect3 id="example-cluster-pgpool-config-backend-settings">
+  <sect3 id="pgpool-config-backend-settings">
    <title>Backend Settings</title>
    <para>
     Specify the <productname>PostgreSQL</productname> backend information.
@@ -863,7 +863,7 @@
    </note>
   </sect3>
 
-  <sect3 id="example-cluster-pgpool-config-online-recovery">
+  <sect3 id="pgpool-config-online-recovery">
    <title>Pgpool-II Online Recovery Configurations</title>
    <para>
     Next, configure the required parameters to perform online recovery.
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/example-replication-si-mode.sgml pgpool-II-4.6.0/doc/src/sgml/example-replication-si-mode.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/example-replication-si-mode.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/example-replication-si-mode.sgml	2025-03-05 04:34:14.754142187 +0000
@@ -71,7 +71,7 @@
    </tgroup>
   </table>
 
-  <table id="example-replication-mode-table-postgresql-config">
+  <table id="mode-table-postgresql-config">
    <title>PostgreSQL version and Configuration</title>
    <tgroup cols="3">
     <thead>
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/loadbalance.sgml pgpool-II-4.6.0/doc/src/sgml/loadbalance.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/loadbalance.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/loadbalance.sgml	2025-03-05 04:30:40.101330096 +0000
@@ -154,7 +154,7 @@
 
  </sect2>
 
- <sect2 id="runtime-config-writing-queries-may-affect-load-balancing">
+ <sect2 id="writing-queries-may-affect-load-balancing">
 
   <title>Writing queries may affect Load Balancing</title>
   <para>
@@ -223,7 +223,7 @@
   </para>
  </sect2>
 
- <sect2 id="runtime-config-load-balancing-in-streaming-replication">
+ <sect2 id="load-balancing-in-streaming-replication">
 
   <title>Load Balancing in Streaming Replication</title>
 
@@ -1123,7 +1123,7 @@
 	 <replaceable>"[object]:[dependent-object]"</replaceable>
      In an explicit transaction block after a WRITE statement has been issues, this will prevent
 	 load balancing of any READ statements containing references of dependent object(s).
-	 <example id="example-dml-adaptive-object-relationship-list-1">
+	 <example id="adaptive-object-relationship-list-1">
 	 <title>Configuring dml adaptive object relationship</title>
 	   <para>
 	    If you have a trigger installed on table_1 that do INSERT in  <literal>table_2</> for each
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/memcache.sgml pgpool-II-4.6.0/doc/src/sgml/memcache.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/memcache.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/memcache.sgml	2025-03-05 04:31:46.637581518 +0000
@@ -586,7 +586,7 @@
   </variablelist>
  </sect2>
 
- <sect2 id="runtime-in-memory-query-cache-memcached-config">
+ <sect2 id="memory-query-cache-memcached-config">
   <title>Configurations to use memcached</title>
 
   <para>
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/online-recovery.sgml pgpool-II-4.6.0/doc/src/sgml/online-recovery.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/online-recovery.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/online-recovery.sgml	2025-03-05 04:31:23.493494057 +0000
@@ -88,7 +88,7 @@
   and <ulink
   url="https://git.postgresql.org/gitweb/?p=pgpool2.git;a=blob_plain;f=src/sample/scripts/recovery_2nd_stage.sample;hb=refs/heads/master">/etc/pgpool-II/recovery_2nd_stage.sample</ulink>.
   Example installation using those scripts can be found in <xref
-  linkend="example-cluster-pgpool-config-online-recovery">.
+  linkend="pgpool-config-online-recovery">.
  </para>
 
  <para>
@@ -149,7 +149,7 @@
       </para>
      </listitem>
     </itemizedlist>
-    The script example can be found in <xref linkend="example-cluster-pgpool-config-online-recovery">.
+    The script example can be found in <xref linkend="pgpool-config-online-recovery">.
      <note>
       <para>
        The script path and filename are hard coded, <command>$PGDATA/pgpool_remote_start</command> is executed on main(primary) node.
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/performance.sgml pgpool-II-4.6.0/doc/src/sgml/performance.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/performance.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/performance.sgml	2025-03-05 04:32:53.103832877 +0000
@@ -216,7 +216,7 @@
    mode, so from now on we focus on the mode.
   </para>
 
-  <sect2 id="session-level-load-balancing-vs-statement-level-load-balancing">
+  <sect2 id="balancing-vs-statement-level-load-balancing">
    <title>Session Level Load Balancing vs. Statement Level Load Balancing</title>
    <para>
     By default load balance mode is "session level" which means
diff -ruN pgpool-II-4.6.0.orig/doc/src/sgml/watchdog.sgml pgpool-II-4.6.0/doc/src/sgml/watchdog.sgml
--- pgpool-II-4.6.0.orig/doc/src/sgml/watchdog.sgml	2025-02-27 07:09:05.000000000 +0000
+++ pgpool-II-4.6.0/doc/src/sgml/watchdog.sgml	2025-03-05 04:32:13.630683520 +0000
@@ -713,7 +713,7 @@
     </listitem>
    </varlistentry>
 
-   <varlistentry id="guc-allow-multiple-failover-requests-from-node" xreflabel="allow_multiple_failover_requests_from_node">
+   <varlistentry id="multiple-failover-requests-from-node" xreflabel="allow_multiple_failover_requests_from_node">
     <term><varname>allow_multiple_failover_requests_from_node</varname> (<type>boolean</type>)
      <indexterm>
       <primary><varname>allow_multiple_failover_requests_from_node</varname> configuration parameter</primary>
