diff --git a/src/test/regression/tests/002.native_replication/PgTester.java b/src/test/regression/tests/002.native_replication/PgTester.java
index e735342..44b77dc 100644
--- a/src/test/regression/tests/002.native_replication/PgTester.java
+++ b/src/test/regression/tests/002.native_replication/PgTester.java
@@ -20,7 +20,7 @@ public class PgTester {
 			Class.forName ("org.postgresql.Driver");
 
 			conn =
-				DriverManager.getConnection("jdbc:postgresql://localhost:11002/test");
+				DriverManager.getConnection("jdbc:postgresql://localhost:11000/test");
 		} catch (ClassNotFoundException e) {
 			e.printStackTrace();
 		} catch (SQLException e) {
@@ -30,7 +30,7 @@ public class PgTester {
 		try {
 			//conn.setAutoCommit(false);
             PreparedStatement query = conn.prepareStatement("INSERT INTO public.sequencetester (recordno) VALUES (?)");
-            int max = startValue + 10;
+            int max = startValue + 200;
             for(int cv = startValue; cv <= max; cv++){
                 query.setInt(1, cv);
                 query.execute();
diff --git a/src/test/regression/tests/002.native_replication/test.sh b/src/test/regression/tests/002.native_replication/test.sh
index 70e8559..16c7aa5 100755
--- a/src/test/regression/tests/002.native_replication/test.sh
+++ b/src/test/regression/tests/002.native_replication/test.sh
@@ -51,14 +51,24 @@ java PgTester 1000 &
 wait
 
 $PSQL -p 11002 test <<EOF
-\copy (SELECT * FROM t1 ORDER BY i) to 'dump0.txt'
+\copy (SELECT * FROM t1 ORDER BY i) to 'dump_t0.txt'
 EOF
 
 $PSQL -p 11003 test <<EOF
-\copy (SELECT * FROM t1 ORDER BY i) to 'dump1.txt'
+\copy (SELECT * FROM t1 ORDER BY i) to 'dump_t1.txt'
+EOF
+
+$PSQL -p 11002 test <<EOF
+\copy (SELECT * FROM sequencetester ORDER BY id) to 'dump_s0.txt'
+EOF
+
+$PSQL -p 11003 test <<EOF
+\copy (SELECT * FROM sequencetester ORDER BY id) to 'dump_s1.txt'
 EOF
 
 # check if database contents are identical
+cat dump_t0.txt dump_s0.txt > dump0.txt 
+cat dump_t1.txt dump_s1.txt > dump1.txt 
 diff dump0.txt dump1.txt
 if [ $? != 0 ];then
 	# contents are not identical
